src/Orient/Orient.Client/Protocol/Connection.cs

```diff
@@ -13,7 +13,7 @@ internal class Connection : IDisposable
         private TcpClient _socket;
         private BufferedStream _networkStream;
         private byte[] _readBuffer;
-        private int RECIVE_TIMEOUT = 30 * 1000; // Recive timeout in milliseconds

```

Comment:
May be it could be kept as `public const int DefaultReceiveTimeoutInMillisec = 30 * 1000;`